hevm.cabal

```diff
@@ -103,8 +104,17 @@ library
     Paths_hevm
   autogen-modules:
     Paths_hevm
-  if os(linux) || os(windows)
+  if impl(ghc >= 9.4) && !os(darwin)
```

Comment:
not sure we need the ghc version condition here. I'm fine with only supporting 9.4+ unless you need older versions for echidna?

Reply:
We actually [just moved echidna to 9.4](https://github.com/crytic/echidna/pull/1096) 😄 so if you'd prefer to only support 9.4+ that's fine by me. It's not a lot of code to keep the 9.2 build-ability though (just the `else` block after this bit)